locklift/ton/index.ts

```diff
@@ -148,16 +168,16 @@ class Ton {
       },
       result: 'balance'
     });
-    
+
     return new BigNumber(balance);
   }
-  
+
   /**
    * Get account type for specific address. Raise an error if address not found
    * @param address
    * @returns {Promise<{acc_type: *, acc_type_name: *}>}
    */
-  async getAccountType(address) {
+  async getAccountType(address: string): Promise<{ acc_type: string; acc_type_name: string }> {
```

Comment:
Let's make a types folder and migrate types here, those types can be used  outside the locklift